### DISCUSSION TITLE
Using all Pins of the actuator connection

### DISCUSSION THREAD

**Discussion by WFeneberg** [Lenti84/VdMot_Controller#82] [Ideas]
My work colleague uses the Vdmot for his heating control. It requires long cable lengths for the actuators. He noticed that only 2 pins of the connections are occupied. He thinks you could use all the pins to increase the wire cross section.

**Comment by SurfGargano** [comment]
The pinning is compatible to the Homematic HMIP-VdMot.
But you can use the other pins , because they are unconnected.
But the resistance of the pins are not the problem.
The problem could be the resistance of the cable and the emc influence from outside.
So use a shielded cable and connected the shield at the VDMot to GND or maybe to PE (only one side of the shield).
I have 10 meters from controller to 2 actors installed along other power lines. It works without problems. 
I used shielded KNX cable ( 4x 0.8 mm²).

**Comment by Lenti84** [comment]
One can also use different connectors to connect thicker wire. 
There are Pads for soldering connectors with 5mm pitch.
For example:
![grafik](https://user-images.githubusercontent.com/72278363/226467179-0859b610-9a64-4825-beb1-2d52e87568c4.png)
